features/storage/kvstore/filesystemstore/FileSystemStore.h

```diff
@@ -0,0 +1,223 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef MBED_FILE_SYSTEM_STORE_H
+#define MBED_FILE_SYSTEM_STORE_H
+
+#include "KVStore.h"
+#include "FileSystem.h"
+
+#ifndef FSST_FOLDER_PATH
+#define FSST_FOLDER_PATH "$fsst$" //default FileSystemStore folder path on fs
```

Comment:
@geky - thanks for the input, we'll remove the "$"
Regarding the config system, if I understood you correctly then its already there.
This is in case configuration does not define it for some reason..
(see mbed-os/features/storage/kvstore/conf/filesystem_no_rbp/mbed_lib.json 
 or   mbed-os/features/storage/kvstore/conf/filesystem/mbed_lib.json )

        "folder_path": {
            "help": "Path for the working directory where the FileSystemStore stores the data",
            "value": "kvstore"
        }

Reply:
There is a parameter in mbed_lib.json "folder_path"